contrib/sbtc/token-key-admin/contracts/sbtc-token-key-admin.clar

```diff
@@ -0,0 +1,126 @@
+
+;; title: sbtc-token-key-admin
+;; version:
+;; summary:
+;; description:
+
+;; traits
+;;
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+
+;; token definitions
+;; 
+(define-fungible-token sbtc u2100000000000000)
+
+;; constants
+;;
+(define-constant contract-owner tx-sender)
+(define-constant err-invalid-caller u1)
+(define-constant err-owner-only u100)
+(define-constant err-not-token-owner u101)
+
+;; data vars
+;;
+(define-data-var coordinator (optional {addr: principal, key: (buff 33)}) none)
+
+;; data maps
+;;
+(define-map signers uint {addr: principal, key: (buff 33)})
+
+;; public functions
+;;
+(define-public (set-coordinator-data (data {addr: principal, key: (buff 33)}))
+    (if (is-valid-caller)
+        (ok (var-set coordinator (some data)))
+        (err err-invalid-caller)
+    )
+)
+
+(define-public (set-signer-data (id uint) (data {addr: principal, key: (buff 33)}))
+    (if (is-valid-caller)
+        (ok (map-set signers id data))
+        (err err-invalid-caller)
+    )
+)
+
+(define-public (delete-signer-data (id uint))
+    (if (is-valid-caller)
+        (ok (map-delete signers id))
+        (err err-invalid-caller)
+    )
+)
+
+(define-public (mint! (amount uint))
+    (if (is-valid-caller)
```

Comment:
`mint` and `burn` should be gated to coordinator using `assert-tx-sender-is-coordinator!`

Reply:
Done.